test/testdata/errcheck_exclude_functions.go

```diff
@@ -0,0 +1,19 @@
+//args: -Eerrcheck
+//config: linters-settings.errcheck.check-blank=true
+//config: linters-settings.errcheck.exclude-functions=io/ioutil.ReadFile,io/ioutil.ReadDir
```

Comment:
could you use a config file instead of the config comment system?

example: https://github.com/golangci/golangci-lint/blob/36aa12425c0aaf5697c2cd7c094f84c410ddd9d7/test/testdata/forbidigo_example.go#L2